src/app/users/users.service.ts

```diff
@@ -25,4 +25,38 @@ export class UsersService {
     return this.http.get('/users');
   }
 
+  /**
+   * @param {any} user User to be created.
+   * @returns {Observable<any>}
+   */
+  createUser(user: any): Observable<any> {
+    return this.http.post('/users', user);
+  }
+
+  /**
+   * @returns {Observable<any>} Offices data.
+   */
+  getOffices(): Observable<any> {
+    return this.http.get('/offices');
+  }
```

Comment:
Remove this.
Fetch from the users template instead, see community app code for reference. (allowed offices and available roles) Make all the necessary changes for this.
https://demo.mifos.io/fineract-provider/api/v1/users/template?tenantIdentifier=default&pretty=true
https://github.com/openMF/community-app/blob/develop/app/scripts/controllers/organization/CreateUserController.js